packager/util/http.go

```diff
@@ -35,10 +35,7 @@ var ConditionalRequestHeaders = map[string]bool{
 var legacyHeaders = map[string]bool{
 	"Connection": true,
 	"Keep-Alive": true,
-	"Proxy-Authenticate": true,
```

Comment:
I assume the duplication you're referring to is `statefulResponseHeaders` in `signer.go`. I think I prefer that duplication, because it's semantically attached to a different concept (RFC2616 vs SXG spec). For instance, if in the future, SXG v=b4 no longer disallows `Proxy-Authenticate`, we still want to remove it here because we're acting as a gateway. Or perhaps in the future, we have a pluggable fetch API; we still want it removed by `signer.go` so it doesn't generate an invalid SXG.

Reply:
Yes, I will fix this.